### DISCUSSION TITLE
How to train model based on a pretrained one?

### DISCUSSION THREAD

**Discussion by DmitriiTrofimov** [explosion/spaCy#10383] [Help: Model Advice] [answered]
I want to take as base the en_core_web_trf and adjust with new corpora to enhance accuracy on my data. How can I do that?

**Comment by DmitriiTrofimov** [comment, marked as answer]
I've found very good example [here](https://github.com/explosion/projects/tree/v3/pipelines/ner_demo_update).

**Reply by polm** [reply]
Glad you found the answer. Note that most of the time it is better to train your own model following the normal process described in the docs rather than doing the update described there - in particular for Transformers you'll still be using the original Transformer weights, so you'll be starting with a lot of info anyway.

**Reply by DmitriiTrofimov** [reply]
Got it. Thanks!
